voltdbclient/client.go

```diff
@@ -0,0 +1,240 @@
+/* This file is part of VoltDB.
+ * Copyright (C) 2008-2016 VoltDB Inc.
```

Comment:
How can we have header but go doc wont include it in documentation? when I run go doc I get these headers.